arch/arm/configs/bcm2709_defconfig

```diff
@@ -928,6 +928,7 @@ CONFIG_VIDEO_ARDUCAM_PIVARIETY=m
 CONFIG_VIDEO_IMX219=m
 CONFIG_VIDEO_IMX258=m
 CONFIG_VIDEO_IMX290=m
+CONFIG_VIDEO_IMX296=m
```

Comment:
Of course now the commit text is wrong ;-)

Reply:
Arghhhhh